modules/multipath/data/org.freedesktop.UDisks2.Multipath.xml

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE node PUBLIC
+"-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+
+<!--
+ Copyright (C) 2015 Gris Ge <fge@redhat.com>
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General
+ Public License along with this library; if not, write to the
+ Free Software Foundation, Inc., 59 Temple Place, Suite 330,
+ Boston, MA 02111-1307, USA.
+-->
+
+  <!-- ******************************************************************* -->
+
+  <!--
+    org.freedesktop.UDisks2.Drive.Multipath:
+    @short_description: Multipath interface exported on disk objects
+  -->
+  <interface name="org.freedesktop.UDisks2.Drive.Multipath">
+    <!-- Multipath:
+        If the disk drive is multipath device, this is set to the
+        org.freedesktop.UDisks2.Multipath object that it corresponds to. It is
+        '/' otherwise.
+    -->
```

Comment:
This is an interesting idea, but we should consider how it would apply to other situations, eg:

- block device is member of an
  - LVM Volume Group
  - MD RAID array/set
  - BIOS/firmware RAID array/set
- block device is backing device for a LUKS encrypted device (udisks probably doesn't show separate device for decrypted LUKS, though)

So all of these are cases in which the block device being examined should not be accessed directly, but rather should be ignored in favor of whatever composite/aggregate device it is part of. Do we want to come up with a generic concept for this instead of having separate properties for all of the above cases? If so, that may be beyond the scope of this pull request.